connect-inject/container_init.go

```diff
@@ -334,8 +334,9 @@ EOF
 export CONSUL_HTTP_ADDR="${HOST_IP}:8500"
 export CONSUL_GRPC_ADDR="${HOST_IP}:8502"
 {{- end}}
-{{- if .AuthMethod }}
 consul-k8s connect-init -acl-auth-method="{{ .AuthMethod }}" \
+  -pod-name=${POD_NAME} -pod-namespace=${POD_NAMESPACE} \
```

Comment:
The trailing `\` here is ugly but is still functional, open to ideas on how to improve it.

Reply:
per our convo on naming here and subsequent changes marking this one resolved